### ISSUE TITLE
Assess the impact of the new LOLE calculation

### ISSUE THREAD

**Issue by ChaelKruip** [quintel/etsource#896]
This includes assessing:

* the impact of subtracting the wind-curves directly from the demand curve instead of using the capacity credit for wind
* the impact of subtracting solar curves from the demand curve (currently not done)
* Checking if we correctly take availability into account for the reliable capacity calculation

See also https://github.com/quintel/etsource/pull/895

**Comment by jorisberkhout** [comment]
I started reviewing the new LOLE calculation and noticed that we still take into account the reliable part of the installed wind capacity in [reliable_electricity_production_capacity.gql](https://github.com/quintel/etsource/blob/master/gqueries/modules/security_of_supply/reliable_electricity_production_capacity.gql), even though we subtract the hourly electricity production by wind turbines from the hourly demand. This explains the large difference between the LOLE on LIVE and BETA for the Urgenda scenario.

I did a quick fix for this issue in my local model which results in much more comparable results for the LOLE in the Urgenda scenario on LIVE and BETA. I will push a more permanent fix soon.

**Comment by ChaelKruip** [comment]
:+1: great catch @jorisberkhout!

**Comment by AlexanderWirtz** [comment]
:whoop:

**Comment by jorisberkhout** [comment]
Ok, I created a group `dispatchable_production` and added all converters with a capacity credit of `1.0` to it (see 6941ecf3f0458dcdb52f3416add12b430e40fee2). I also excluded all solar electricity producer (PV and CSP) and hydro river from the LOLE calculation (see 78ffe2b426d9d08ae76f0d5ff7b1e293d8019304).

The new LOLE calculation now works as I expect it to, although it does give different results from the old calculation. One major point still open is chart 111. This chart shows the reliable and unreliable electricity production capacity and is still based on the capacity credit and hence has no link to the new LOLE calculation. Furthermore it is the only place where we still need the capacity credit, so I would be very much on favour of changing this chart such that we can (finally) remover the `capacity_credit` attribute and all associated area attributes and gqueries on ETSource. I will discuss potential solutions with @ChaelKruip .
